src/catalogue/category/property/allowedValuesListTextFields.component.tsx

```diff
@@ -42,6 +42,7 @@ const AllowedValuesListTextFields = (props: {
 
   const { fields, append, remove } = useFieldArray({
     control,
+    // Sets the correct name for the field array based on the type of `nestIndex` and the presence of 'properties' in `propertyRHF`.
```

Comment:
Could you just clarify when this is the case, I can see what the comment says  by reading the line, what isn't so clear is why its nested. I guess just something like `When multiple properties are given allowed values will be nested within those properties` if I am understanding it correctly.

Reply:
Isn't that implied by the type anyways. The comment would just be describing the parent type form either (AddPropertyMigration | AddCatalogueCategoryWithPlacementIds).

Reply:
True, but I didn't see that first and to me that is less obvious than the comment as it is now which is just describing the if statement rather than why the if statement was needed.

Reply:
I've removed the comment and I don't think there is a need for one as name is determined by the type given to the useForm function in react hook form anyways.

Reply:
Sure its clearer once you know what its doing, which is what a comment is supposed to help you understand, but in this case I think it was my fault, it may be clearer to others.